### DISCUSSION TITLE
Let the user set the date priorization

### DISCUSSION THREAD

**Discussion by chronikm** [photoprism/photoprism#3005] [Ideas]
Hi,
first of all thank you very much for this great software!

However, I keep running in one problem: If I edit a video file (e.g. cut out the boring parts of a long drone video), I make sure that the new video file has a xmp sidecar with all the same information as the original video, especially when it comes to date information. My usual workflow is to either do this with digikam or with some other software packages to transfer the meta data from the original video to the new one. 

Date fields that are easily changed by this workflow are:

- xmp:ModifyDate
- exif:DateTimeDigitized
- exif:DateTimeOriginal

Date fields that are a little more extensive (in my workflow) but work as well in one go via digikam are:

- photoshop:DateCreated
- xmp:CreateDate
- xmp:MetadataDate
- video:DateTimeDigitized
- video:ModificationDate
- video:DateUTC
- tiff:DateTime

However, the one field that is the most annoying is a date that sticks within the MP4 file and seems to be chosen by photoprism. On my German Windows machine it is called **"Medium erstellt" (media created)** within the **"Ursprung" (origin)** section of the file properties. I can change this property with exiftool via "FileCreateDate", but this always leads to a long copy and replace activity as exiftool cannot change this property directly but has to create a new file with a changed property. Also it cannot be added into my usual workflow easily.

Some time ago I ran across some photoprism code lines that seem to set the priority of different date values but I cannot find this source anymore. Would it be possible to let the user change the priority of the date fields you already handle in your code? Or is it even already possible by modifying some properties file?

Best regards and thanks again for this great piece of software!

**Comment by graciousgrey** [comment]
Thank you very much for your kind feedback!

Unfortunately, it is not possible to change the prioritization at the moment.

You can find the implemented priorities here: 
   https://github.com/photoprism/photoprism/blob/develop/internal/meta/data.go

For more information, see also: https://docs.photoprism.app/user-guide/library/metadata/
